OpenTabletDriver.UX/Windows/Tablet/TabletDebugger.cs

```diff
@@ -171,13 +175,26 @@ protected override async void OnClosing(CancelEventArgs e)
 
             await App.Driver.Instance.SetTabletDebug(false);
 
-            dataRecordingOutput?.Close();
-            dataRecordingOutput = null;
+            stopRecording();
+            reportsRecordedGroup.Visible = false;
         }
 
         private Label deviceName, rawTablet, tablet, reportRate, reportsRecorded;
         private TabletVisualizer tabletVisualizer;
-        private CheckBox enableDataRecording;
+
+        private readonly Button startDataRecordingButton = new Button
+        {
+            Text = "Start Recording",
```

Comment:
```suggestion
            Text = "⏵Start Recording",
```

I considered adding a unicode triangle to the start recording button (and a unicode square symbol for the stop recording button) - but was unsure if this would display correctly on all platforms, or if it'd look right.

@InfinityGhost thoughts?

Reply:
After some experimentation it seems that unicode symbols aren't universal, and many platforms cannot display them, or use emojis instead - so cancel this idea.